project/Dependencies.scala

```diff
@@ -6,7 +6,7 @@ object Dependencies {
   val workbenchGoogle2Version = s"0.32-${workbenchLibsHash}"
   val workbenchAzureVersion = s"0.5-${workbenchLibsHash}"
   val openTelemetryVersion = s"0.6-${workbenchLibsHash}"
-  val doobieVersion = "1.0.0-RC2"
+  val doobieVersion = "1.0.0-RC4"
```

Comment:
Is this why you shifted away from the HikariTransactor above?

Reply:
kind of..we're still using `HikariTransactor` in the actual cron job code. It's just I'm making the code to use `Transactor` in test helpers so that in tests, we can use a different one for simplicity